foia_hub/settings/notdev.py

```diff
@@ -13,3 +19,16 @@
         },
     }
 }
+
+AWS_STORAGE_BUCKET_NAME = os.getenv('FOIA_S3_STATIC_BUCKET_NAME')
+AWS_S3_CUSTOM_DOMAIN = 's3.amazonaws.com/%s' % AWS_STORAGE_BUCKET_NAME
+STATIC_URL = 'https://%s/' % AWS_S3_CUSTOM_DOMAIN
+STATICFILES_STORAGE = 'storages.backends.s3boto.S3BotoStorage'
+
+# Don't add complex authentication related query parameters for requests
+AWS_QUERYSTRING_AUTH = False
+
+try:
```

Comment:
These next few lines need go back into staging.py and production.py. We want this override to happen last.